### ISSUE TITLE
Staging Review finding: Labels likely to be missed by screen reader users on results cards

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93697]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

One of the common navigation techniques for screen reader users when arriving on a page is to explore the heading structure, skipping heading-to-heading th skim the page. When they hear a heading that sounds like what they're looking for, they then have their screen reader announce the content of the page starting at that heading.

In the results cards, the label that appears in the top left-hand corner of the card comes before the heading in the DOM. If a screen reader user decides to listen to the contents of that card starting at the heading, they won't hear the label at all.

This isn't super critical, since the content of the card is understandable and actionable without that label. But the labels help sighted users understand the category of beneift immediately when viewing the card, and screen reader users should get the same immediate information to decide if this is what they're looking for.

<details><summary>Link, screenshot or steps to recreate</summary>

Example of a benefit card:

![GI Bill education benefit card, with text describing the program and two action links: Learn more and Apply now.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82863-Brian-2.png?raw=true)

</details>

## Recommended action

Easy solution here! Group the label inside the H3 heading for each card:
```
<h3>
   <span class="usa-label">Education</span>
   <span class="some-other-classes">GI Bill</span>
</h3>
```
You may need some extra CSS to match the existing appearance for sighted users. But for screen reader users, this ensures that the label is announced as part of the heading that describes the card.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Confirmed fixed and tested in Sprint 14.
